applications/openshift/general/general_apply_scc/rule.yml

```diff
@@ -21,11 +21,14 @@ severity: medium
 ocil_clause: 'SCCs in Pod definitions need review'
 
 ocil: |-
-   Review the pod definitions in your cluster and verify that you have security
-   contexts defined as appropriate.  OpenShift's Security Context Constraint
-   feature is on by default in OpenShift 4 and applied to all pods deployed. SCC
-   selection is determined by a combination of the values in the securityContext
-   and the rolebindings for the account deploying the pod.
+   Review the pod definitions in your cluster and verify they have appropriate
+   security contexts. OpenShift comes configured with default security context
+   constraints you can use immediately to secure pods in your cluster. For more
+   information on security context constraints, how to use them, and how to
+   build your own, please refer to the OpenShift security constraints
+   documentation [1].
+
+   [1] https://docs.openshift.com/container-platform/4.11/authentication/managing-security-context-constraints.html
```

Comment:
Please use the `weblink` macro for this. Instead of this style, I may suggest doing some like the following:
```
 ...,please refer to the OpenShift security constraints documentation, {{{ weblink(link="https://docs.openshift.com/container-platform/4.11/authentication/managing-security-context-constraints.html" }}}.
```

[Example](https://github.com/ComplianceAsCode/content/blob/2b2152d288e05f0d64f26fff3f01b0e75311023d/linux_os/guide/services/http/securing_httpd/httpd_modules_improve_security/httpd_deploy_mod_security/group.yml)

Reply:
That's handy. Thanks!